Source/qol/chatlog.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file chatlog.h
+ *
+ * Adds Chat log QoL feature
+ */
+#pragma once
+
+#include "engine.h"
+#include "player.h"
+
+namespace devilution {
+
+extern bool ChatLogFlag;
+
+void ToggleChatLog();
+void AddMessageToChatLog(const std::string &message, Player *player = nullptr, UiFlags flags = UiFlags::ColorWhite);

```

Comment:
Maybe use a string_view instead of string so that you don't have to create a string from const char* when calling this function.

Reply:
Then I have to do 
![image](https://user-images.githubusercontent.com/14297035/156310705-404456b4-d0cf-4e73-8d4a-65219a74dda5.png)
in AddMessageToChatLog anyway